src/components/20-molecules/datepicker/index.scss

```diff
@@ -80,7 +80,7 @@ axa-datepicker {
 
 .m-datepicker {
   position: relative;
-  display: inline;
+  display: block;
```

Comment:
This needs to be tested in a form

Reply:
Looks quite nice: https://axa-ch.github.io/plib-feature/bugfix/1864_datepicker_display/?path=/story/components-datepicker-demos--in-a-form

Reply:
Or what to check in a form?

Reply:
Aem forms. Just to make sure is ok there

Reply:
Have changed it at chrome-dev-tools at acc stage at a aem form. Do not change anything there. Is that test ok?

Reply:
At IE on aem forms is looks better because extra space on top of datepicker is gone.